service/templates/patch_version.html

```diff
@@ -56,7 +56,7 @@
         </div>
         <div class="col-xs-4">
           <div class="row">
-            <div class="btn btn-lg btn-primary" ng-class="[[patch.Version.length > 0 ? 'btn-xs' :'btn-lg']]" ng-click="save()" ng-disabled="selectionCount().numTasks==0">
+            <div class="btn btn-lg btn-primary" ng-class="[[patch.Version.length > 0 ? 'btn-xs' :'btn-lg']]" ng-click="save()" ng-disabled="selectionCount().numTasks==0 || disableSubmit">
```

Comment:
Put disableSubmit first, so that if it's true selectionCount() doesn't need to run.